### PR TITLE
Added Subitems on the Details task page

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/163697/wrike-new-features

### CODE REVIEW

File: src/components/common/SubItem/SubItem.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+export type Props = {
+  item: ITaskFromList;
+  onComplete?: (
+    itemId: ITaskFromList["id"],
+    complete: boolean,
+  ) => Promise<unknown>;
+};
+
+const SubItem: FC<Props> = ({ item, onComplete }) => {
+  const boxSize = 14;
```
Comment (HappyPaul55) [inline src/components/common/SubItem/SubItem.tsx:16]: Is this because there is nothing in `@deskpro/ui`?

File: src/constants.ts [hunk 2]
```diff
@@ -29,3 +29,5 @@ export const CustomFieldsMap = {
 };
 
 export const IMPORTANCES = ["High", "Normal", "Low"];
+
+export const STATUSES = ["Active", "Completed", "Deferred", "Cancelled"];
```
Comment (HappyPaul55) [inline src/constants.ts:33]: Should this be an `enum`

**Review by HappyPaul55 — APPROVED** [review]

File: src/constants.ts (hunk 2 above)
Comment (zpawn) [reply src/constants.ts:33]: Deleted. I didn’t notice after the refactoring that it wasn’t being used anywhere.

File: src/components/common/SubItem/SubItem.tsx (hunk 1 above)
Comment (zpawn) [reply src/components/common/SubItem/SubItem.tsx:16]: it's to make it look nice. By default, in `deskpro-ui` it’s 12px

Screenshot 2024-09-18 at 17 46 02

**Pull request merged**